components/kyma-environment-broker/internal/model.go

```diff
@@ -62,6 +62,17 @@ type Instance struct {
 	DeletedAt time.Time
 }
 
+func (instance Instance) GetProvisioningParameters() (ProvisioningParameters, error) {
+	var pp ProvisioningParameters
+
+	err := json.Unmarshal([]byte(instance.ProvisioningParameters), &pp)
+	if err != nil {
+		return pp, errors.Wrap(err, "while unmarshaling provisioning parameters")
```

Comment:
Does it make sense to return `nil` instead of `pp` in case of error?
`return nil, errors.Wrap(err, "while unmarshaling provisioning parameters")`.

Reply:
It already equals `nil` because of the var declaration

Reply:
Returning `nil, err` is more explicit and readable. Otherwise, we need to trace the code above to make sure that `pp` is nil and remains nil.

Reply:
But pp is a `struct` value not a pointer

Reply:
You're correct, then let's leave it like that.